bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -71,6 +73,7 @@
   private ExecutorService bufferedMutatorExecutorService;
 
   private BigtableSession session;
+  private BigtableApi bigtableApi;
 
```

Comment:
would it make sense to make `bigtableApi` final? May be as well as `session`

Reply:
Thanks for the review!! 

yes, it would.